tools/README.md

```diff
@@ -0,0 +1,79 @@
+# Run Airflow3 Locally
+
+This guide walks you through setting up Apache Airflow 3 locally using Hatch and a Postgres container. You'll run a Postgres container to use it as a backend database for Airflow and set up the necessary environment to run Airflow.
```

Comment:
@pankajastro Airflow 2 standalone would work okay with Sqlite locally. Did this change? I am curious about the benefits/downsides of introducing this Postgres dependency at this point for local testing purposes.

Reply:
SQLite works, I just find Postgres more stable, shall I make the Postgres setup steps optional and keep SQLite as default?